lib/src/screens/buy/onramper_page.dart

```diff
@@ -70,17 +63,30 @@ class OnRamperPageBody extends StatefulWidget {
 class OnRamperPageBodyState extends State<OnRamperPageBody> {
   OnRamperPageBodyState();
 
-  @override
-  void initState() {
-    super.initState();
-    if (Platform.isAndroid) WebView.platform = SurfaceAndroidWebView();
-  }
-
   @override
   Widget build(BuildContext context) {
-    return WebView(
-        initialUrl: widget.uri.toString(),
-        backgroundColor: widget.backgroundColor,
-        javascriptMode: JavascriptMode.unrestricted);
+    return InAppWebView(
+      initialOptions: InAppWebViewGroupOptions(
+        crossPlatform: InAppWebViewOptions(transparentBackground: true),
+      ),
+      initialUrlRequest: URLRequest(
+        url: Uri.tryParse(
+          widget.uri.toString(),
+        ),
+      ),
```

Comment:
`widget.uri` is already a `URI` why convert to string and then parse to URI again